config/routes.rb

```diff
@@ -1,20 +1,24 @@
 Rails.application.routes.draw do
 
-  root to: 'home#index'
-  resources :cards
-  resources :reviews, only: [:new, :create]
-  resources :decks do
-    get :set_current, on: :member
+  root to: 'guest/home#index'
+  scope module: "dashboard" do
+    resources :cards
+    resources :reviews, only: [:new, :create]
+    resources :decks do
+      get :set_current, on: :member
+    end
+    resource  :profile, controller: :profile, only: [:edit,:update]
+    delete 'logout' => 'user_sessions#destroy', :as => :logout
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.